### DISCUSSION TITLE
Showing DOWN ARROW on homepage

### DISCUSSION THREAD

**Discussion by chinabiue** [squidfunk/mkdocs-material#7308] [Q&A] [answered]
Does any one know how to enable this "arrow down" on home page in setting?
I try '-- navigation.down' but seems didn't work. 
Thanks in advanced if you have the answer.

![arrow](https://github.com/squidfunk/mkdocs-material/assets/2224440/980b712a-2423-48fc-8688-1c98c8c3aba3)

**Comment by alexvoss** [comment, marked as answer]
That will be a customization for the documentation homepage since it is related to the parallax effect. There have been [discussions about building a similar effect](https://github.com/squidfunk/mkdocs-material/discussions?discussions_q=parallax). It is a customization, through, and the particular one used on the Material for MkDocs homepage is quite a sophisticated one. Building it not only involved a good deal of CSS skills but also design skills.
